CMakeLists.txt

```diff
@@ -422,8 +422,12 @@ if(WIN32)
 
     file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/conf/" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/conf/")
     file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
-    file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/README.md" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
+    #file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/README.md" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
```

Comment:
Why the comment, should we just remove this line?

Reply:
I guess the thinking behind this was that if this is ever changed, the README needs special handling. But I can certainly just remove the line, I don't really have a strong argument against it.

Reply:
removed the line in [13090d3](https://github.com/apache/nifi-minifi-cpp/pull/1709/commits/13090d370c39c1713add87d037cb388b3d8b4307)